samestr/summarize/summarize.py

```diff
@@ -59,19 +53,15 @@ def summarize(args):
                                                    similarity_threshold=args['aln_pair_min_similarity'])
 
     # merge all data, replace NaN with 0 only in the shared_strains and analyzed_strain column
-    cooc_data = pd.merge(mp_cooc, strain_cooc, on=['row', 'col'], how='outer')
+    cooc_data = pd.merge(taxon_cooc, strain_cooc, on=['row', 'col'], how='outer')
     cooc_data['analyzed_strain'] = cooc_data['analyzed_strain'].fillna(0)
     cooc_data['shared_strain'] = cooc_data['shared_strain'].fillna(0)
     cooc_data.iloc[:, 2:] = cooc_data.iloc[:, 2:].mask(
         cooc_data.iloc[:, 2:].isna(), other=np.nan).astype('Int64')
 
     # write output
-    mp_taxonomy.to_csv(args['output_dir'] +
-                       '/mp_taxonomy.tsv', sep='\t', index=False)
-    mp_species.drop(['kingdom', 'phylum', 'class', 'order', 'family', 'genus'], axis=1).to_csv(
-        args['output_dir'] + '/mp_species.tsv', sep='\t', index=False)
-    mp_counts.to_csv(args['output_dir'] +
-                     '/mp_counts.tsv', sep='\t', index=False)
+    taxon_counts.to_csv(args['output_dir'] +
+                     '/taxon_counts.tsv', sep='\t', index=False)
```

Comment:
```suggestion
    taxon_counts.to_csv(os.path.join(args['output_dir'],
                     'taxon_counts.tsv', sep='\t', index=False)
```

If you use `os.path.join`, you don't need to manually attach `/` everywhere